library/src/test/java/kwasm/format/text/TableTypeTest.kt

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kwasm.format.text
+
+import com.google.common.truth.Truth.assertThat
+import kwasm.format.ParseContext
+import kwasm.format.ParseException
+import org.junit.Assert.assertThrows
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+@RunWith(JUnit4::class)
+@UseExperimental(ExperimentalUnsignedTypes::class)
+class TableTypeTest {
+
+    private val context = ParseContext("TokenizerTest.wasm", 1, 1)
+    private val tokenizer = Tokenizer()
+
+    @Test
+    fun parseSingleNumber_withoutFuncref_throwsParseException() {
+        val expectedMin = 123456.toUInt()
+        val tokens = tokenizer.tokenize("$expectedMin", context)
+        val exception = assertThrows(ParseException::class.java) { tokens.parseTableType(0) }
+        assertThat(exception).hasMessageThat().contains("Too few arguments")
```

Comment:
"Too few arguments" is a bit vague. I think "Expected 'funcref'" would be fine here.